### DISCUSSION TITLE
Blank Page on Storage Tab in UI for Spark Jobs

### DISCUSSION THREAD

**Discussion by Nikhilpa1** [NVIDIA/spark-rapids#9503] [Q&A]
When I run a Spark job, everything seems to be working fine, but when I navigate to the Storage tab in the UI (not history), I see a blank page.
I've checked my Spark configuration and logs, but I can't seem to pinpoint the cause of this issue. Has anyone else encountered a similar problem, or does anyone have suggestions on how to debug and resolve this?

**Comment by tgravescs** [comment]
Can you provide more details?  What version of Spark, what cluster manager are you using?   You are caching data or have run something that you are expecting something to be in the storage tab?    it only shows information if it has something to show.
